### PR TITLE
fix: add create transaction with grpc #NEBD-105

### PR DESCRIPTION
**Pull request by pritamghanghas**
**[Issue](https://nebraltd.atlassian.net/browse/NEBD-105)**

- Link: https://nebraltd.atlassian.net/browse/NEBD-105
- Summary:

**How**

**Screenshots**

**References**

**Checklist**

- [ ] Tests added
- [ ] Cleaned up commit history (rebase!)
- [ ] Documentation added
- [ ] Thought about variable and method names

### CODE REVIEW

File: gatewayconfig/bluetooth/characteristics/add_gateway_characteristic.py [hunk 1]
```diff
@@ -40,7 +65,7 @@ def StartNotify(self):
 
         self.PropertiesChanged(constants.GATT_CHRC_IFACE, {"Value": self.notifyValue},
                                [])
-        self.add_timeout(30000, self.AddGatewayCallback)
+        # self.add_timeout(30000, self.AddGatewayCallback)
```
Comment (shawaj) [inline gatewayconfig/bluetooth/characteristics/add_gateway_characteristic.py:68]: Do we need to leave the commented code in?

File: gatewayconfig/bluetooth/characteristics/add_gateway_characteristic.py [hunk 2]
```diff
@@ -52,32 +77,27 @@ def WriteValue(self, value, options):
...
+            # back. Don't see much value in extra complexity.
         except Exception:
             logger.exception("Unable to register gateway for unknown reason")
 
     def ReadValue(self, options):
         logger.debug('Read Add Gateway')
+        value = self.create_add_gateway_txn()
+        # logger.debug(f"value from grpc call: {value}")
+        self.notifyValue = string_to_dbus_byte_array(value)
+        # logger.debug(f"converted {self.notifyValue}")
```
Comment (shawaj) [inline gatewayconfig/bluetooth/characteristics/add_gateway_characteristic.py:95]: Do we need all this commented out code?

**Review by shawaj — CHANGES_REQUESTED** [review]

File: gatewayconfig/bluetooth/characteristics/add_gateway_characteristic.py (hunk 2 above)
Comment (pritamghanghas) [reply gatewayconfig/bluetooth/characteristics/add_gateway_characteristic.py:95]: not really, as the code is untested. wanted to keep it for debugging.

Comment (shawaj) [reply gatewayconfig/bluetooth/characteristics/add_gateway_characteristic.py:95]: np

Comment (pritamghanghas) [reply gatewayconfig/bluetooth/characteristics/add_gateway_characteristic.py]: removed now.

File: gatewayconfig/bluetooth/characteristics/add_gateway_characteristic.py (hunk 1 above)
Comment (pritamghanghas) [reply gatewayconfig/bluetooth/characteristics/add_gateway_characteristic.py]: removed.

**Review by shawaj — APPROVED** [review]
LGTM

**Pull request merged**